src/EditorFeatures/CSharpTest/CodeActions/ConvertLocalFunctionToMethod/ConvertLocalFunctionToMethodTests.cs

```diff
@@ -299,13 +299,52 @@ void M(int i, int j)
     delegate int D(int a, ref string b);
     void M(int i, int j)
     {
-        D x = (int a, ref string b) => LocalFunction1(a, ref b, ref i, j);
+        var x = (D)((int a, ref string b) => LocalFunction1(a, ref b, ref i, j));
```

Comment:
it's unclear to me that these changes are better than what we had before.  Casts of lambdas seem much harder to read, vs just having the type on the left.

Reply:
we keep the original code. this is just to make sure that we parenthesize the lambda.
(anywhere else we should it?)

Reply:
oh, i see.  yes. this is fine :)